scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
```

Comment:
:stop_sign: **Security Vulnerability**: Path traversal vulnerability detected. This allows access to files outside the intended directory[^3].
```suggestion
// Use path.join and validate the resulting path
const safePath = path.join('/tmp', filename)
if (safePath.startsWith('/tmp/')) {
    fs.readFileSync(safePath)
} else {
    throw new Error('Invalid file path')
}
```